plugin.audio.radiothek/resources/lib/Addon.py

```diff
@@ -220,6 +220,26 @@ def getSearchHistory(pluginhandle):
             add_directory(search_query, "", "", "", "", search_query, "search_detail", pluginhandle)
 
 
+def getAudioQuality():
+    quality_list = ['q1a', 'q2a', 'q3a', 'q4a', 'qxb']
+    audio_protocol = getAudioProtocol()
+    if audio_protocol == 'shoutcast':
+        quality_index = int(xbmcaddon.Addon().getSetting('audioQuality'))

```

Comment:
Maybe in the future consider an accessor method to reuse the same instance

Reply:
Also we have methods on the API to get the settings values as int, no need to cast it on the python side